scripts/proposicoes/destaques/process_destaques.R

```diff
@@ -63,9 +74,12 @@ process_proposicoes_destaques <- function(
               by = c("id_leggo")) %>%
     left_join(proposicoes_criterio_mais_comentadas_twitter,
               by = c("id_leggo")) %>%
+    left_join(proposicoes_criterio_requerimento_urgencia,
+              by = c("id_leggo")) %>% 
     mutate(criterio_aprovada_em_uma_casa = !is.na(criterio_aprovada_em_uma_casa),
            criterio_avancou_comissoes = !is.na(criterio_avancou_comissoes),
            criterio_pressao_alta = !is.na(criterio_pressao_alta),
+           requerimento_urgencia_apresentado = !is.na(requerimento_urgencia_apresentado, requerimento_urgencia_aprovado),
```

Comment:
Aqui diz que is.na(x) só recebe uma coluna por vez, aí uma opção seria substituir por:

> requerimento_urgencia_apresentado = !is.na(requerimento_urgencia_apresentado) | !is.na(requerimento_urgencia_aprovado)

(se for um "|" mesmo)